src/twisted/web/_flatten.py

```diff
@@ -272,7 +272,7 @@ def keepGoing(
         dataEscaper: Callable[[Union[bytes, str]], bytes] = dataEscaper,
         renderFactory: Optional[IRenderable] = renderFactory,
         write: Callable[[bytes], object] = write,
-    ) -> Generator[Union[Generator, Deferred[Generator]], None, None]:
+    ) -> Generator[Union[Flattenable, Deferred[Flattenable]], None, None]:
```

Comment:
Kinda weird that this worked before!

Reply:
I just spent 20 minutes trying to figure out why it worked before, and my best guess now is "mypy bug."